src/main/java/ch/njol/skript/expressions/ExprAI.java

```diff
@@ -0,0 +1,77 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.expressions;
+
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.classes.Changer;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.coll.CollectionUtils;
+
+@Name("AI")
+@Description("Returns whether an entity has AI.")
+@Examples("set artificial intelligence of target entity to false")
+@Since("INSERT VERSION")
+public class ExprAI extends SimplePropertyExpression<LivingEntity, Boolean> {
+	
+	static {
+		register(ExprAI.class, Boolean.class, "(ai|artificial intelligence)", "livingentities");
+	}
+	
+	@Override
+	public Class<? extends Boolean> getReturnType() {
+		return Boolean.class;
+	}
+	
+	@Override
+	protected String getPropertyName() {
+		return "artificial intelligence";
+	}
+	
```

Comment:
The method order, as the conventions file states, is:
 (init) -> convert -> (acceptChange) -> (change) -> getReturnType -> getPropertyName.